tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1058,7 +1058,40 @@ static struct device_info boards[] = {
 		.first_sysupgrade_partition = "os-image",
 		.last_sysupgrade_partition = "file-system",
 	},
+	/** Firmware layout for the Archer A6 v3  */
+	{
+		.id     = "ARCHERA6-V3",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+			"{product_name:Archer A6,product_ver:3.0.0,special_id:55530000}\n"
+			"{product_name:Archer A6,product_ver:3.0.0,special_id:54570000}\n"
+			"{product_name:Archer C6,product_ver:3.0.0,special_id:55530000}\n",
```

Comment:
That's not A6, that's C6. If you want to support both devices, this should be mentioned somewhere, or one should even have two images for them.